### PR TITLE
Fix calendar android build

### PR DESCRIPTION
**Pull request by murilopereirame**
Adapts the bump-version script to bump the version of apps according to their language (Groovy for old gradle and kotlin for kts). Moves some functions that are used in both App and Worker.

### CODE REVIEW

File: buildSrc/bump-version.js [hunk 1]
```diff
@@ -76,12 +77,17 @@ async function bumpIosVersion(newVersionString) {
...
+ * @param buildGradlePath {string}
  * @return {Promise<void>}
  */
-async function bumpAndroidVersion(currentVersion, newVersionString) {
-	const buildGradlePath = "app-android/app/build.gradle"
+async function bumpAndroidVersion(currentVersion, newVersionString, buildGradlePath) {
 	const buildGradleString = await fs.promises.readFile(buildGradlePath, "utf8")
-	const oldVersionCodeMatch = buildGradleString.match(/versionCode (\d+)/)
+
+	const kotlinRegex = /versionCode = (\d+)/
```
Comment (charlag) [inline buildSrc/bump-version.js:86]: just a thought: we should put it somewhere in gradle.properties or something, where it's easier to parse

File: android.js [hunk 2]
```diff
@@ -80,22 +80,22 @@ async function buildAndroid({stage, host, buildType, existing, webClient, app})
...
-		await $`rm -r build/app-android`
+		await $`rm -r ${buildDir}/app-android`
 	} catch (e) {
 		// Ignoring the error if the folder is not there
 	}
 
 	const appTarget = app === "mail" ? "app" : "calendar"
 	const {version} = JSON.parse(await $`cat package.json`.quiet())
-	const apkName = `${app === "mail" ? "tutanota" : "calendar"}-tutao-${buildType}-${version}.apk`
+	const apkName = `${app}-tutao-${buildType}-${version}.apk`
```
Comment (charlag) [inline android.js:92]: I would not rename mail app apk, this might backfire somewhere

File: src/common/serviceworker/CommonMailWorker.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
+// @bundleInto:common
```
Comment (charlag) [inline src/common/serviceworker/CommonMailWorker.ts:1]: 1. the name is very misleading. Should `CommonMailUtils` be `MailUtils` (or `SharedMailUtils`) and this one be `CommonMailUtils`?
2. Why is it in service worker directory of all places? Why is it not in `common/src/api/common` where "common" (as in: common between main thread & worker) code resides?

**Review by charlag — CHANGES_REQUESTED** [review]
Please go over commits and apply the formatting so that we don't have an extra commit.

I also noticed that `/src/mail-app/mail/model/MailUtils.ts` and `src/mail-app/mail/MailUtils.ts` are both empty? we should clean them up

**Comment by charlag** [comment]
Also we discussed fixed chunking rules so that worker cannot depend on gui transitively

**Comment by murilopereirame** [comment]
> Also we discussed fixed chunking rules so that worker cannot depend on gui transitively

What do you mean by fixed chunking rules? Didn't we decided to just move the functionality stuff to main and move what is common and needed by the worker to a common file?

File: src/common/api/common/CommonMailUtils.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
+// @bundleInto:common
```
Comment (charlag) [inline src/common/api/common/CommonMailUtils.ts:1]: you probably don't need the annotation anymore?

**Review by charlag — APPROVED** [review]

File: src/common/api/common/CommonMailUtils.ts (hunk 4 above)
Comment (murilopereirame) [reply src/common/api/common/CommonMailUtils.ts]: No, I don't. Removed

**Pull request merged**